core/document/src/layers/style/mod.rs

```diff
@@ -56,6 +56,24 @@ impl PathStyle {
 	pub fn new(stroke: Option<Stroke>, fill: Option<Fill>) -> Self {
 		Self { stroke, fill }
 	}
+	pub fn get_fill(&self) -> Option<Fill> {
```

Comment:
please remove the get wherever you see it: https://rust-lang.github.io/api-guidelines/naming.html
our codebase is not yet consistent in that regard but it is something we want to improve on